src/main/java/com/sourceclear/marvin/Marvin.java

```
@@ -122,6 +131,31 @@ public Model fetchMavenPomFile(String groupId, String artifactId, String version
 
 
   /**
+   * Gets the license.txt file if it exists
+   * @param groupId the group id
+   * @param artifactId the artifact id
+   * @param version  the version
+   * @return an input stream of the file
+   */
+  public InputStream fetchLicenseTxt(String groupId, String artifactId, String version) {
+    try {
+      String url = createMavenLicenseFileFromId(groupId, artifactId, version);
+      HttpResponse<InputStream> licenseFile = Unirest.get(url).asBinary();
+
+      if (licenseFile.getStatus() != HttpStatus.SC_OK) {
+        return null;
```

i think in terms of a library API, it seems like it would be good to:

* output some error here, and/or
* throw an IOException instead of returning null

otherwise if I'm a client, i have to add a null check, instead of just catching an exception which would be cleaner.